src/pages/api/identities/index.ts

```diff
@@ -55,25 +55,27 @@ const getHandler: Handler = (request, response): Promise<void> => {
       })
       .catch((error) => {
         span.setDisclosedAttribute("identities found", false);
+
         if (error instanceof GraphqlErrors) {
           throw webErrorFactory({
             ...webErrors.identityNotFound,
             parentError: error,
           });
         }
+
         if (error instanceof ConnectUnreachableError) {
           throw webErrorFactory({
             ...webErrors.connectUnreachable,
             parentError: error,
           });
         }
+
         throw error;
       });
 
     span.setDisclosedAttribute("identities found", true);
     response.statusCode = HttpStatus.OK;
-    response.json(identities);
-    return;
+    return response.end(JSON.stringify(identities));
   });
```

Comment:
@Fenntasy This fixes the `API resolved without sending a response for /api/identities/, this may result in stalled requests.` false positive. On the client side, this does not change anything, as SWR will serialise the data fetched.

What do you think?